### PR TITLE
*: make IsVirtualGeneratedColumn more readable

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close None
Problem Summary:

### What is changed and how it works?

just clean code.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by time-and-fate — APPROVED** [review]

**Review by windtalker — APPROVED** [review]
LGTM

**Review by XuHuaiyu — APPROVED** [review]

File: ddl/column.go [hunk 1]
```diff
@@ -1396,7 +1396,7 @@ func (w *updateColumnWorker) calcChecksums() []uint32 {
 			w.checksumBuffer.Cols = w.checksumBuffer.Cols[:0]
 		}
 		for _, col := range w.table.DeletableCols() {
-			if col.ID == id || (col.IsGenerated() && !col.GeneratedStored) {
+			if col.ID == id || (col.IsVirtualGeneratedColumn()) {
```
Comment (tangenta) [inline ddl/column.go:1399]:
```suggestion
			if col.ID == id || (col.IsVirtualGenerated()) {
```

File: parser/model/model.go [hunk 2]
```diff
@@ -129,6 +129,14 @@ type ColumnInfo struct {
 	Version uint64 `json:"version"`
 }
 
+// IsVirtualGeneratedColumn checks the column if it is virtual.
+func (c *ColumnInfo) IsVirtualGeneratedColumn() bool {
+	if c.IsGenerated() && !c.GeneratedStored {
+		return true
+	}
+	return false
```
Comment (tangenta) [inline parser/model/model.go:137]:
```suggestion
	return c.IsGenerated() && !c.GeneratedStored
```

Comment (hawkingrei) [reply parser/model/model.go]: Done.

File: ddl/column.go (hunk 1 above)
Comment (hawkingrei) [reply ddl/column.go]: Done

**Pull request merged**